dev/storage.h

```diff
@@ -907,9 +907,9 @@ namespace sqlite_orm {
              *  ```
              */
             template<class... Args>
-            void insert(Args... args) {
+            int64 insert(Args... args) {
```

Comment:
there is one interesting feature in SQLite called `RETURNING` https://www.sqlite.org/lang_returning.html. @trueqbit let's imagine how we are going to implement this. E.g. this query is expected to return `void`
```c++
storage.insert(into<User>(), 
    columns(&User::id, &User::name),
    values(std::make_tuple(1, std::string("Ellie"))));
```
But this query
```c++
auto insertedUser = storage.insert(into<User>(), 
    columns(&User::id, &User::name),
    values(std::make_tuple(1, std::string("Ellie"))),
    returning(asterisk());
```
is expected to return  a `User`. And this query
```c++
auto insertedId = storage.insert(into<User>(), 
    columns(&User::id, &User::name),
    values(std::make_tuple(1, std::string("Ellie"))),
    returning(&User::id);
```
is expected to return inserted id **no matter what type id has**. And `RETURNING` feature will just erase this diff. I know I did not tell you about this feature idea but know I am doing this and looks like I am doing this right in time.